bindings/experimental/distrdf/python/DistRDF/Profiling/Flamegraph.py

```diff
@@ -0,0 +1,97 @@
+from functools import partial
```

Comment:
Check that all files that use type hints also have `from __future__ import annotations` at the beginning